Makefile.js

```diff
@@ -580,6 +580,19 @@ target.test = function() {
 
     target.webpack();
 
+    const browserFileLintOutput = new CLIEngine({
```

Comment:
I don't feel strongly about this, but would it be simpler/less time-consuming to just pull in espree here and do a parse? I don't understand what checks might be needed in the linting when we're not using any configuration.

Reply:
That's true. I had been thinking running the linter would be useful if there were other ES5-only checks we wanted to enable in addition to checking for ES5 syntax, but since no such checks are actually enabled, using `espree` on its own would probably be simpler.

As a separate PR, I wonder if it would be worthwhile to have a special-case optimization in `Linter` that skips traversing the AST when there are no rules enabled. That seems like it would make the performance of "ESLint with no rules enabled" similar to the performance of espree.